unity/Assets/Maroon/scenes/experiments/ThrowingBalls/scripts/MaxCameraPosition.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MaxCameraPosition : MonoBehaviour, IResetObject
+{
+    private static MaxCameraPosition _instance;
+    private Vector3 _startPosition;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        _startPosition = transform.position;
+    }
+
+    public void ChangePosition()
+    {
+        transform.position = new Vector3(9.7f, 7.1f, 12.7f);
```

Comment:
is it a good idea to assign this position hardcoded?
and maybe use the CoordCameraController for changing the position for max and min
I think we can remove the MaxCameraPosition and MinCameraPosition script in this case